### DISCUSSION TITLE
fastcgi_finish_request()

### DISCUSSION THREAD

**Discussion by eroltoker** [brefphp/bref#1863] [Support]
This doesn't appear to be supported in bref-php?

if (function_exists('fastcgi_finish_request')) {
    fastcgi_finish_request();
} else {
    Logger::warn('doesn't exist);
}

returns the log 'doesn't exist'

1) Is that expected
2) Is there a workaround?

The goal is to return a response to the client and then do some delayed processing, pushing data to a data warehouse middleware, which doesn't have a fast response time)

Thanks in advance!

**Comment by mnapoli** [comment]
Hi, this is not supported (https://bref.sh/docs/use-cases/http#no-fastcgi_finish_request), you have to use a queue + worker: https://bref.sh/docs/use-cases/sqs

**Reply by eroltoker** [reply]
okay, that was my backup.   Sorry i didn't catch this in the docs.  Thank you!
